src/System.Runtime.Extensions/ref/Configurations.props

```diff
@@ -1,9 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
System.Runtime.Extensions should not have package configurations as it doesn't build a package. Just leave it as it was -- using BuildConfigurations